hack.py

```diff
@@ -5,22 +5,36 @@
 @author: Quantum Liu
 """
 
-import requests
+from __future__ import print_function

```

Comment:
You don't need to import print_function
python2 and 3 can use print as function.